coalib/parsing/DefaultArgParser.py

```diff
@@ -87,6 +77,20 @@ def default_arg_parser(formatter_class=None):
                             action='version',
                             version=Constants.VERSION)
 
+    mode_group = arg_parser.add_argument_group('Mode')
+
+    mode_group.add_argument(
+        '--ci', const=True, action='store_const',
+        help="run coala in non interactive mode")
+
+    mode_group.add_argument(
+        '--json', const=True, action='store_const',
+        help="mode in which coala dumps output as json")
+
+    mode_group.add_argument(
+        '--format', const=True, nargs='?', metavar='STR',
+        help="mode in which coala dumps output as csv")
```

Comment:
any idea on how that help message could look? for me it looks fugly, should be something like "display output as custom format string", right?